### PR TITLE
Nc news 11 articles topic query

### PR DESCRIPTION
**Pull request by KashaM767**

### CODE REVIEW

File: __tests__/seed.test.js [hunk 1]
```diff
@@ -75,6 +75,29 @@ describe('GET /api/articles', () => {
     });
 });
 
+describe('GET /api/articles/?topic', () => {
+    test('200: filters the articles by the topic value specified in the query.', () => {
```
Comment (nc-poppy-m) [inline __tests__/seed.test.js:79]: One more 200 test needed - what happens if we get a topic which exists but has no articles?

File: models/comments.js [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+const format = require('pg-format');
 
-exports.checkExists = (article_id) => {
-    return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
-        .then(({ rows }) => {
-            if (!rows.length) {
-                return Promise.reject({ status: 404, msg: "not found" });
-            }
-        })
+exports.checkExists = (table, column, value) => {
```
Comment (nc-poppy-m) [inline models/comments.js:4]: Nice idea to have a reusable checkExists!

**Review by nc-poppy-m — CHANGES_REQUESTED** [review]
Nearly there! You just need one more 200 test.

**Review by nc-mick-fay — APPROVED** [review]
Great extra testing - You can merge into main when you're ready!

**Pull request merged**